tools/api-markdown-documenter/package.json

```diff
@@ -71,7 +71,7 @@
 		"@fluid-tools/markdown-magic": "file:../markdown-magic",
 		"@fluidframework/build-common": "^2.0.3",
 		"@fluidframework/build-tools": "^0.26.1",
-		"@fluidframework/eslint-config-fluid": "^2.0.0",
+		"@fluidframework/eslint-config-fluid": "file:../../common/build/eslint-config-fluid",
```

Comment:
Note for reviewers: this is the primary change. The rest of the changes are to address new rules that are being pulled in.